src/DriverUtilities.ts

```diff
@@ -12,17 +12,60 @@ import { Capabilities, Builder, By, until, WebDriver } from 'selenium-webdriver'
 import https = require('https');
 import fetch from 'node-fetch';
 const firefox = require('selenium-webdriver/firefox');
+const chrome = require('selenium-webdriver/chrome');
+
+require('geckodriver');
+require('chromedriver');
+
+interface DriverSettings {
+    headless: Boolean;
+}
+
+const defaultSettings: DriverSettings = {headless: false};
```

Comment:
We should default to headless as true, tests don't run very well when not running in headless on the build infrastructure

Reply:
typo, sure should be headless only